WooCommerce/Classes/Extensions/Product+SwiftUIPreviewHelpers.swift

```diff
@@ -0,0 +1,72 @@
+#if canImport(SwiftUI) && DEBUG
```

Comment:
I still not used the SwiftUI Preview, so maybe I'm wrong, but is it really needed this optional import here?

Reply:
this conditional check isn't necessary for this `Product` extension, but I thought it's better making it only available for SwiftUI preview in debug mode since this helper is not used in the app 🙂